packages/office-addin-dev-settings/test/unit-tests/test.ts

```diff
@@ -115,6 +118,26 @@ describe("DevSettingsForAddIn", function() {
         const expected = new devSettings.SourceBundleUrlComponents(undefined, undefined, undefined, undefined);
         await testSetSourceBundleUrlComponents(actual, expected);
       });
+      it("webView can be set to Default", async function() {
+        await devSettings.setWebView(addinId, devSettings.WebViewType.Default);
+        assert.strictEqual(await devSettingsWindows.getWebView(addinId), "default");
+      });
+      it("webView can be set to IE", async function() {
+        await devSettings.setWebView(addinId, devSettings.WebViewType.IE);
+        assert.strictEqual(await devSettingsWindows.getWebView(addinId), "ie");
+      });
+      it("webView can be set to Edge", async function() {
+        await devSettings.setWebView(addinId, devSettings.WebViewType.Edge);
+        assert.strictEqual(await devSettingsWindows.getWebView(addinId), "edge");
+      });
+      it("webView can be set to Edge Chromium", async function() {
+        await devSettings.setWebView(addinId, devSettings.WebViewType.EdgeChromium);
+        assert.strictEqual(await devSettingsWindows.getWebView(addinId), "edge chromium");
+      });
+      it("webView can be cleared", async function() {
```

Comment:
You don't need Clear. Setting default is clearing the value. Have I mentioned this enough times?  :-)